db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/CreateConstraintConstantAction.java

```diff
@@ -302,7 +302,7 @@ public void	executeConstantAction( Activation activation )
 			case DataDictionary.FOREIGNKEY_CONSTRAINT:
 				ReferencedKeyConstraintDescriptor referencedConstraint = DDUtils.locateReferencedConstraint
 					( dd, td, constraintName, columnNames, otherConstraintInfo );
-				DDUtils.validateReferentialActions(dd, td, constraintName, otherConstraintInfo,columnNames);
+				// DDUtils.validateReferentialActions(dd, td, constraintName, otherConstraintInfo,columnNames);
```

Comment:
Is `com.splicemachine.db.impl.sql.execute.CreateConstraintConstantAction#executeConstantAction` reachable? if so, how? if not, I suggest we remove it.

It looks almost identical to `com.splicemachine.derby.impl.sql.execute.actions.CreateConstraintConstantOperation#executeConstantAction`

Reply:
... to be removed, follow up in [DB-10689](https://splicemachine.atlassian.net/browse/DB-10689).